unity-renderer/Assets/DCLFeatures/ScreencaptureCamera/CameraObject/Scripts/ScreenRecorder.cs

```diff
@@ -20,6 +24,88 @@ public ScreenRecorder(RectTransform canvasRectTransform)
+
+            // Set the base camera's target texture
+            RenderTexture originalTargetTexture = baseCamera.targetTexture;
+            baseCamera.targetTexture = initialRenderTexture;
+
+            // Iterate through all overlay cameras and set their target textures
+            var overlayCameras = baseCameraData.cameraStack;
+            var originalOverlayTargetTextures = new List<RenderTexture>();
+
+            if (overlayCameras != null)
+            {
+                foreach (Camera overlayCamera in overlayCameras)
+                {
+                    originalOverlayTargetTextures.Add(overlayCamera.targetTexture);
+                    overlayCamera.targetTexture = initialRenderTexture;
+                }
+            }
+
+            // Render the cameras to the RenderTexture
+            baseCamera.Render();
+
+            baseCamera.targetTexture = originalTargetTexture;
+
+            if (overlayCameras != null)
+            {
+                // Revert the target textures of the base and overlay cameras
+                for (var i = 0; i < overlayCameras.Count; i++)
+                    overlayCameras[i].targetTexture = originalOverlayTargetTextures[i];
+            }
+
+            // Create an intermediate RenderTexture with the size of the cropped area
+            RenderTexture finalRenderTexture = RenderTexture.GetTemporary(targetScreenFrame.ScreenWidthInt, targetScreenFrame.ScreenHeightInt, 0);
+            Graphics.Blit(initialRenderTexture, finalRenderTexture);
+
+            // // Apply any post-processing effects or other operations using Graphics.Blit
+            // Graphics.Blit(initialRenderTexture, intermediateRenderTexture);
+            //
+            // // Create a Texture2D with the same GraphicsFormat as the RenderTexture
+            // Texture2D screenshot = new Texture2D(width, height, TextureFormat.RGBAHalf, false);
+            // Graphics.CopyTexture(intermediateRenderTexture, screenshot);
+            //
+            // Camera.main.targetTexture = null;
+            // RenderTexture.ReleaseTemporary(initialRenderTexture);
+            // RenderTexture.ReleaseTemporary(intermediateRenderTexture);
+            //
+            // // Define the specific rect you want to copy
+            // int rectX = 100;
+            // int rectY = 100;
+            // int rectWidth = 500;
+            // int rectHeight = 500;
+            //
+            // // Create a Texture2D with the same GraphicsFormat as the RenderTexture and the size of the rect
+            // Texture2D screenshot = new Texture2D(rectWidth, rectHeight, TextureFormat.RGBAHalf, false);
+            //
+            // // Copy the specific rect from the intermediateRenderTexture to the screenshot Texture2D
+            // Graphics.CopyTexture(intermediateRenderTexture, 0, 0, rectX, rectY, rectWidth, rectHeight, screenshot, 0, 0, 0, 0);
```

Comment:
Can we remove all this commented code?

Reply:
sry @sandrade-dcl , this one should be a draft.... Another one get merged (mentioned in history)